### PR TITLE
Fix typos and docs standards

### PR DESCRIPTION
**Pull request by akkspros**
fix typos and docs standards

### CODE REVIEW

File: documentation-standards.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Here's an example of the PHPdoc annotating the `wp cron event schedule` command:
  * : The hook name.
  *
  * [<next-run>]
- * : A Unix timestamp or an English textual datetime description compatible with `strtotime()`. Defaults to now.
+ * : A Unix timestamp or an English textual date time description compatible with `strtotime()`. Defaults to now.
```
Comment (schlessera) [inline documentation-standards.md]: Here, `datetime` refers to a type of data (date + time in one single scalar value). This is a commonly used term in software engineering.

Please revert this particular change.

File: bin/command.php [hunk 2]
```diff
@@ -382,15 +382,15 @@ private static function gen_cmd_pages( $cmd, $parent = array(), $verbose = false
 			$docs = $cmd['longdesc'];
 			$docs = htmlspecialchars( $docs, ENT_COMPAT, 'UTF-8' );
 
-			// decrease header level
+			// decrease header level.
```
Comment (schlessera) [inline bin/command.php]: While we're at it, we can also capitalize the first word of the comments to comply with WPCS.
```suggestion
			// Decrease header level.
```

File: bin/command.php [hunk 3]
```diff
@@ -382,15 +382,15 @@ private static function gen_cmd_pages( $cmd, $parent = array(), $verbose = false
...
 			$docs = preg_replace( '/^## /m', '### ', $docs );
 
-			// escape `--` so that it doesn't get converted into `&mdash;`
+			// escape `--` so that it doesn't get converted into `&mdash;`.
```
Comment (schlessera) [inline bin/command.php]:
```suggestion
			// Escape `--` so that it doesn't get converted into `&mdash;`.
```

File: bin/command.php [hunk 4]
```diff
@@ -417,7 +417,7 @@ private static function gen_cmd_pages( $cmd, $parent = array(), $verbose = false
 			}
 			$docs = implode( "\n", $bits );
 
-			// hack to prevent double encoding in code blocks
+			// hack to prevent double encoding in code blocks.
```
Comment (schlessera) [inline bin/command.php]:
```suggestion
			// Hack to prevent double encoding in code blocks.
```

**Comment by schlessera** [comment]
Thanks for the PR, @akkspros. There are a few minor changes needed before I can merge this.

**Comment by akkspros** [comment]
@schlessera can you check this please

**Pull request merged**